spin-redirecter/package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "spin-version-proxy",
+  "name": "spin-redirector",
```

Comment:
The directory and the Wasm file are is (incorrectly) spelled `redirecter` - should we be consistent?  It seems like the Wasm file name doesn't match the package name anyway...!